### ISSUE TITLE
rowStyle overwritten on rows with .z-odd class

### ISSUE THREAD

**Issue by andrewekeren** [zippyui/react-datagrid#59]
```javascript
tableRowStyle: function(data, props) {
                var style = {};
                if (data.type == 'Parent') {
                    style.background = '#d9dbdc';
                    style.fontWeight = 'bold';
                }
                else {
                    style.background = '#fff';
                }
                return style;
            }
```

Font weight displays correctly, but the background displays as #f2f7ff from .z-odd.

Tested on Chrome 43.0.2357.81.

**Comment by radubrehar** [comment]
Thanks for reporting.
We've just released `1.2.0` on npm which should fix that. Sorry for the delay.

The problem was that the background of odd/even rows was actually set in row cells (in css) so applying this to the row had no effect. Should be fixed now on `1.2.0`

**Comment by radubrehar** [comment]
Can you try the latest `1.2.13`?
